x-pack/test/api_integration/apis/metrics_ui/constants.ts

```diff
@@ -19,6 +19,10 @@ export const DATES = {
     },
   },
   '8.0.0': {
+    hosts_only: {
+      min: 1641081600000, // 2022-01-02T00:00:00.000Z
+      max: 1641081930000, // 2022-01-02T00:05:30.000Z,
```

Comment:
Nit: how about this to avoid repeating in the comment?
```suggestion
      min: new Date("2022-01-02T00:00:00.000Z").getTime(),
      max: new Date("2022-01-02T00:05:30.000Z").getTime(),
```